### ISSUE TITLE
Media query inside legacy selector duplicates the legacy selector

### ISSUE THREAD

**Issue by raynor85** [raynor85/postcss-media-legacy#1]
For example this code:

```
.lt-ie9 {
    .select-menu {
        select {
            @media (width >= $bp-sm) {
                padding: 0 0 0 0;
            }
        }
}
```

Generates this incorrect css snippet:

```
.lt-ie9 .lt-ie9 .select-menu select {
    padding: 0;
}
```

**Comment by raynor85** [comment]
Fixed in v1.2.0

**Comment by raynor85** [comment]
Fixed in v1.2.0
